app/models/manageiq/providers/hawkular/middleware_manager/refresh_parser.rb

```diff
@@ -48,6 +48,10 @@ def get_middleware_servers
 
       def alternate_machine_id(machine_id)
         # see BZ #1294461 : here, we'll try to adjust the machine ID to the format from that bug
+        # we expect to get a string like this: 2f68d133a4bc4c4bb19ecb47d344746c
+        # for such string, we should return this: 33d1682f-bca4-4b4c-b19e-cb47d344746c
+        # the actual BIOS UUID is probably returned in upcase, but other providers store it in downcase, so, we let
```

Comment:
it depends on the vendor iirc. I wouldn't write it here anyway because it is "probably" and not for sure.